app/src/lib/api.ts

```diff
@@ -2263,6 +2267,11 @@ export async function isGitHubHost(url: string) {
     return false
   }
 
+  // github.example.com,
+  if (/(^|\.)(github)\./.test(hostname)) {
+    return false
```

Comment:
Shouldn't this be…? 🤔 

```suggestion
    return true
```

Reply:
Eesh, that would have been real bad. Nice catch!